src/lib/backend_fapi.c

```diff
@@ -10,6 +10,96 @@
 FAPI_CONTEXT *fctx = NULL;
 unsigned maxobjectid = 0;
 
+#ifndef FAPI_3_0
+/* The following piece of code is here to enable a preview of the FAPI backend on systems with a
+   TSS < 3.0, and retain the current "FAPI_PREVIEW" feature. */
```

Comment:
Thier has never been a release with FAPI_PREVIEW and even if their was, it was always a developer preview. I see no reason to keep the old behavior.

Reply:
Are you sure ?

Reply:
Yes, things on master can change at any time. The only thing I'll be a huge stickler on is data upgrades. We can never trash user data. But FAPI_PREVIEW was always billed as experimental and use at your own risk. Once we land a release with the feature as a first class citizen, we always need to preserve data.